modules/local/ensemblvep/main.nf

```diff
@@ -0,0 +1,52 @@
+process ENSEMBLVEP {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda (params.enable_conda ? "bioconda::ensembl-vep=104.3" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/ensembl-vep:105.0--pl5321h4a94de4_1' :
+        'quay.io/biocontainers/ensembl-vep:105.0--pl5321h4a94de4_1' }"
+
+    input:
+    tuple val(meta), path(vcf)
+    val   genome
+    val   species
+    val   cache_version
+    path  cache
+
+    output:
+    tuple val(meta), path("*.ann.vcf"), emit: vcf
+    path "*.summary.html"             , emit: report
+    path "versions.yml"               , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def dir_cache = cache ? "\${PWD}/${cache}" : "/.vep"
+    """
+    mkdir $prefix
+
+    vep \\
+        -i $vcf \\
+        -o ${prefix}.ann.vcf \\
+        $args \\
+        --assembly $genome \\
+        --species $species \\
+        --cache \\
+        --cache_version $cache_version \\
+        --dir_cache $dir_cache \\
+        --fork $task.cpus \\
+        --vcf \\
+        --stats_file ${prefix}.summary.html
```

Comment:
We could add the option `--compress_output bgzip` as according to recommendations we should try to compress the output whenever possible. You would also have to change the out file format to `*.ann.vcf.gz`